### PR TITLE
add rules,question card and global contex

### PR DESCRIPTION
**Pull request by Maianki**
## Added rules , question card and global context
Things you will find in this PR
- Rules card
- Question card
- Global context

Thanks for reviewing PR

### CODE REVIEW

File: src/context/global-context.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  useEffect(() => {
+    const quizzesCollectionRef = collection(db, "quizzes");
+    (async () => {
+      try {
+        const data = await getDocs(quizzesCollectionRef);
+        const allQuizzes = data.docs.map((doc) => ({
+          ...doc.data(),
+          id: doc.id,
+        }));
+        console.log(allQuizzes);
```
Comment (BalajiHarinath) [inline src/context/global-context.js]: Can remove this console log

**Review by BalajiHarinath — APPROVED** [review]
Everything looks good to merge. Nothing much to suggest.

File: src/context/global-context.js (hunk 1 above)
Comment (Maianki) [reply src/context/global-context.js]: sure thanks for pointing it out.

**Review by vrx29 — APPROVED** [review]
Everything looks good. Great that you have used firebase as well. you can merge it.

**Comment by Maianki** [comment]
Thanks for reviewing @BalajiHarinath @vrx29

**Pull request merged**